utils/frame/benchmarking-cli/src/block/bench.rs

```diff
@@ -0,0 +1,240 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sc_block_builder::BlockBuilderApi;
+use sc_cli::Result;
+use sc_client_api::{BlockBackend, UsageProvider};
+use sp_api::{ApiExt, BlockId, HeaderT, ProvideRuntimeApi};
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{traits::Block as BlockT, DigestItem};
+
+use clap::Args;
+use log::{info, warn};
+use serde::Serialize;
+use std::{marker::PhantomData, sync::Arc, time::Instant};
+
+use crate::storage::record::Stats;
+
+/// Parameters to configure a block benchmark.
+#[derive(Debug, Default, Serialize, Clone, PartialEq, Args)]
+pub struct BenchmarkParams {
+	/// Skip benchmarking NO-OP extrinsics.
+	#[clap(long)]
+	pub skip_extrinsic: bool,
+
+	/// Skip benchmark an empty block.
+	#[clap(long)]
+	pub skip_block: bool,
```

Comment:
feeling these names are not specific enough :)

`skip_block` especially

Reply:
I renamed them multiple times but I agree that they are still bad :smile: